packages/devtools_app/lib/src/flutter/controllers.dart

```diff
@@ -55,6 +53,25 @@ class ProvidedControllers implements DisposableController {
     );
   }
 
+  /// Builds the default providers for offlineMode.
+  ///
+  /// All screens that support offline imports should have their controller in
+  /// this factory and in [offlineLookup].
+  factory ProvidedControllers.offline() {
+    return ProvidedControllers(
+      logging: null,
+      timeline: TimelineController(),
+      memory: null,
+      performance: null,
+      debugger: null,
+      bannerMessages: null,
+    );
+  }
+
+  Map<String, OfflineControllerMixin> get offlineLookup => {
```

Comment:
this seems redundant with the ProvidedControllers factory constructor for offline which provides the same data.